pkg/kritis/admission/admission.go

```diff
@@ -91,6 +92,16 @@ func handlePod(ar *v1beta1.AdmissionReview, admitResponse *v1beta1.AdmissionRevi
 	return nil
 }
 
+func handleReplicaSet(ar *v1beta1.AdmissionReview, admitResponse *v1beta1.AdmissionReview) error {
+	glog.Info("handling replica set...")
```

Comment:
As an operator, I don't think I would find this message useful unless it mentioned which replica set it is handling. I suggest either adding more context, or using glog.V(2).Info.

Reply:
for sure, added more info to this error and the ones for deployment/pods